opensearch-operator/pkg/reconcilers/configuration.go

```diff
@@ -67,6 +67,11 @@ func (r *ConfigurationReconciler) Reconcile() (ctrl.Result, error) {
 		r.reconcilerContext.AddConfig("plugins.security.system_indices.enabled", "true")
 		r.reconcilerContext.AddConfig("plugins.security.system_indices.indices", string(systemIndices))
 
+		if len(r.instance.Spec.General.AdditionalConfig) > 0 {
```

Comment:
Thanks @cwperks even with this change, if we have to add the `plugins.security.audit.type` with `additionalConfig` will it create 2 values of `plugins.security.audit.type` ?

Reply:
Would it better to add it under `spec.security`? 
```
spec:
  security:
       auditType: "log4j"
  ```

Reply:
Adding @swoehrl-mw

Reply:
No, it won't create 2 files. It will overwrite the default and [log that out](https://github.com/opensearch-project/opensearch-k8s-operator/blob/b80b606964dccb0dd18d1ea01d0c0815eaf0be68/opensearch-operator/pkg/reconcilers/reconcilers.go#L89-L91) accordingly.

Reply:
@cwperks can should be able to get this in https://github.com/opensearch-project/opensearch-k8s-operator/issues/1011, can you update the readme with some details https://github.com/opensearch-project/opensearch-k8s-operator/blob/main/docs/userguide/main.md.
Thanks
@getsaurabh02